dev/unix/tests/release-script.bats

```diff
@@ -0,0 +1,109 @@
+# test the release.sh script
+
+# load the functions from the script
+# (everything before the '### END FUNCTIONS' line)
+eval "$(cat dev/unix/release.sh | sed '/### END FUNCTIONS/q')"
```

Comment:
Using `eval` here, because `source`-ing the file will execute all of the code after the functions, which is no good

Reply:
Would it be better to extract the functions into a "shared library", to avoid having to `sed` scripts?

Reply:
Eventually I plan to put these functions into the install script, and will source that from this script, so that both scripts are using the same functions (and the install script can be downloaded and run individually). I'm currently working on that.

For now, I found a way to detect if the script is being sourced, and return if that is the case (adapted from https://stackoverflow.com/questions/2683279/how-to-detect-if-a-script-is-being-sourced)